src/Core/Providers/ConsoleServiceProvider.php

```diff
@@ -96,7 +97,8 @@ class ConsoleServiceProvider extends ServiceProvider
         'ThemeInstall' => 'command.theme.install',
         'VendorPublish' => 'command.vendor.publish',
         'Serve' => 'command.serve',
-        'WidgetMake' => 'command.widget.make'
+        'WidgetMake' => 'command.widget.make',
+        'RequestMake' => 'command.request.make',
```

Comment:
The code is ok for me. Just that the command lists are organized in alphabetical order in this file. Could you move the `RequestMake` command in the list?

Reply:
of course, no problem